### PR TITLE
Fixed test TCID7 by adding an explicit wait

### PR DESCRIPTION
**Pull request by andreea-qa**

### CODE REVIEW

File: AutomationResources/WebDriverFactory.cs [hunk 1]
```diff
@@ -110,16 +110,16 @@ public IWebDriver CreateSauceDriver()
             //caps.SetCapability("version", "62.0");
             //caps.SetCapability("screenResolution", "1024x768");
 
-            var capabilities =  new DesiredCapabilities();
+            var capabilities =  new ChromeOptions();
             //capabilities.SetCapability(CapabilityType.BrowserName, "chrome");
-            capabilities.SetCapability(CapabilityType.Version, "48.0");
-            capabilities.SetCapability(CapabilityType.Platform, "Linux");
-            capabilities.SetCapability("username", 
+            capabilities.AddAdditionalCapability(CapabilityType.Version, "48.0");
```
Comment (nadvolod) [inline AutomationResources/WebDriverFactory.cs]: @andreea-qa instaed of 48 we can do "latest" so that it's never outdated

**Review by nadvolod — COMMENTED** [review]
@andreea-qa just one tiny update, otherwise really great! I noticed that you have a bunch of commits with every PR. You might need to `git push master` into your own remote branch. I created this almost great diagram and I think maybe you just need to do this step? Good for future reference.
Screen Shot 2020-12-15 at 7 41 06 PM

**Comment by nadvolod** [comment]
@andreea-qa there seem to be some merge conflicts. Are you able to resolve these?

**Comment by andreea-qa** [comment]
> @andreea-qa there seem to be some merge conflicts. Are you able to resolve these?

Hi, there are no conflicts, just a check failed on the CI, not sure if it's something missing on my side or something from the pipeline

**Pull request merged**